### PR TITLE
fix(telecom.phone): set a height for tiles item

### PR DESCRIPTION
**Pull request by fchabrolle**
Signed-off-by: Fabien Chabrolle <fabien.chabrolle@corp.ovh.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-6663
| License          | BSD 3-Clause

fix display of phone tiles

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/choice/choice.less [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .telecom-telephony-line-phone-order {
   .phone-item {
     margin: 8px 0;
+    height: 300px;
```
Comment (radireddy) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/choice/choice.less:6]: why 300px? why not 100%? let it occupy the height it needs.

Comment (fchabrolle) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/choice/choice.less:6]: 100% doesn't fix the issue some tiles don't have the same content, so they're smaller.

**Review by Heiiji — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/choice/choice.less (hunk 1 above)
Comment (antleblanc) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/choice/choice.less:6]: Probably `min-height` could be safer here but the best option we could figure out is that replacing actual grid by using flex.
Could be handle into a separate Pull Request.

**Review by antleblanc — APPROVED** [review]
Thank you @fchabrolle for your contribution.

**Pull request merged**